src/Runner.Sdk/RunnerWebProxy.cs

```diff
@@ -104,8 +104,8 @@ public RunnerWebProxy()
                 _httpsProxyAddress = proxyHttpsUri.AbsoluteUri;
```

Comment:
can you delete `_httpProxyAddress` since it's no longer used?

Reply:
That still gets used via the `HttpsProxyAddress` property in a couple places. Would it make more sense to set `_httpsProxyAddress = httpsProxyAddress;` instead of setting just the environment variables to it?